distro/wildfly/src/main/resources/overlay/standalone/configuration/apiman.properties

```diff
@@ -20,6 +20,7 @@ apiman-gateway.jdbc.jndi-location=java:jboss/datasources/apiman-gateway
 # The following are settings for using elasticsearch for various
 # apiman components.
 # ---------------------------------------------------------------------
+#apiman.es.client.type=es
```

Comment:
If I am not wrong this will also change the quickstart back to Elasticsearch?

Reply:
No, this is just a comment so that if the user uncomments that ES configuration section it will work correctly -- it simply now uses a variable instead of repeating "jest" which is deprecated or doesn't work. 

It's a bit clearer to see in the full diff -- https://github.com/apiman/apiman/pull/973/files/94d760b27e00001ef02a9313ae25f7e3bee64a84#diff-e46e7e0fd0cbfc0b46a9b62be423440ddad0945d451078028cde91ceb75585fe

Reply:
Ahh yes, I did not recognize that these lines are all comments 🚀